### PR TITLE
Re-added recipe for phonenumbers

### PR DESCRIPTION
**Pull request by pmlandwehr**
Github seems to have eaten the feedstock that would have been generated by this merge, so pushing it again.

### CODE REVIEW

File: recipes/phonenumbers/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{% set version = "7.4.2" %}
+package:
+  name: phonenumbers
```
Comment (jakirkham) [inline recipes/phonenumbers/meta.yaml]: Would recommend using `python-` prefixing here.

**Comment by jakirkham** [comment]
cc @patricksnape @ocefpaf (sorry guys moved again)

File: recipes/phonenumbers/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    - phonenumbers.carrierdata
+    - phonenumbers.data
+    - phonenumbers.geodata
+    - phonenumbers.shortdata
+    - phonenumbers.tzdata
+
+about:
+  home: https://github.com/daviddrysdale/python-phonenumbers
+  license: Apache 2.0
+  summary: "Python version of Google's common library for parsing, formatting, storing and validating international phone numbers."
```
Comment (jakirkham) [inline recipes/phonenumbers/meta.yaml]: Is there a shorter summary that we can think of?

File: recipes/phonenumbers/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,39 @@
+{% set version = "7.4.2" %}
+package:
```
Comment (jakirkham) [inline recipes/phonenumbers/meta.yaml]: nit: please add a newline before `package`

**Comment by jakirkham** [comment]
Couple of comments. Let's see what others think.

**Comment by ocefpaf** [comment]
Latest changes in conda-build are making me rethink the python- prefix. Not
sure we should add it unless there is an ambiguity.
Em 13/06/2016 22:09, "jakirkham" <notifications@github.com> escreveu:

File: recipes/phonenumbers/meta.yaml (hunk 2 above)
Comment (pmlandwehr) [inline recipes/phonenumbers/meta.yaml]: Could swap
> Python version of Google's common library
for
> Python port of Google's library

Could also drop "international".  What's the optimal length?

**Comment by pmlandwehr** [comment]
Pypi lists the package as ["phonenumbers"](https://pypi.python.org/pypi/phonenumbers/7.4.2), which is why I don't use the prefix.

**Comment by ocefpaf** [comment]
Ping @pmlandwehr is this good to go?

**Comment by pmlandwehr** [comment]
Yeah, I think so, if the `phonenumbers` vs. `python-phonenumbers` discussion is resolved.

**Comment by ocefpaf** [comment]
> Yeah, I think so, if the `phonenumbers` vs. `python-phonenumbers` discussion is resolved.

We need to re-discuss adding the `python-` prefix in light of the new changes in `conda`. Conda devs are adding namespace support to avoid clashes.

I would add the prefix only in a clear case of name clash, which I am not sure exist here (as the lib which this is based is named `libphonenumber` and not `phonenumber`).

**Pull request merged**